content/docs/container/animated-dock.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: Animated Dock
+description: A sleek dock-style navigation bar, inspired by macOS, that combines glassmorphic design with functionality. With smooth animations and responsive icons, it enhances navigation for a modern web application.
+author: Eshan Singh
```

Comment:
it should be your twitter username @R0X4R

Reply:
Okay let me add my twitter username